pkg/provider/handlers/functions.go

```diff
@@ -62,11 +63,10 @@ func GetFunction(client *containerd.Client, name string) (Function, error) {
 				f.pid = task.Pid()
 
 				// Get container IP address
-				ip, getIPErr := GetIPfromPID(int(task.Pid()))
```

Comment:
Why this change?

Reply:
To have all errors as `err`. I think it's easier visually.